src/main/java/org/nitishm/javacollectionassignment/MongoDB.java

```diff
@@ -0,0 +1,77 @@
+package org.nitishm.javacollectionassignment;
+
+import com.mongodb.BasicDBObject;
+import com.mongodb.DB;
+import com.mongodb.DBCollection;
+import com.mongodb.DBCursor;
+import com.mongodb.MongoClient;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class MongoDB implements UserDB {
+    private MongoClient mongoClient;
+    private DBCollection dbCollection;
+    private String url;
+    private int port;
+    private String database;
+    private String collection;
+
+    public MongoDB(String url, int port, String database, String collection) throws Exception {
+        this.url = url;
+        this.port = port;
+        this.database = database;
+        this.collection = collection;
+        getConnection();
+    }
+
+    private void getConnection() throws Exception {
+        mongoClient = new MongoClient(url, port);
+        DB db = mongoClient.getDB(database);
+        System.out.println("Connect to database successfully");
+        dbCollection = db.getCollection(collection);
+    }
+
+    public List<User> getListOfUsers() throws Exception {
+        DBCursor cursor = dbCollection.find();
+        BasicDBObject obj;
```

Comment:
why have you declared this here?
why not declare in the while loop at line #42?